docs/absolute.md

```diff
@@ -242,6 +242,10 @@ The `duration` argument can be any value that could be passed to `Temporal.Durat
 - any object with properties denoting a duration, such as `{ hours: 5, minutes: 30 }`;
 - a string in ISO 8601 duration format, such as `PT5H30M`.
 
+The `years` and `months` fields of `duration` must be zero, because adding a year or a month to a `Temporal.Absolute` is invalid and will throw a `RangeError`.
+`Temporal.Absolute` is independent of time zones and calendars, and so years and months may be different lengths.
+If you need to do this, convert the `Temporal.Absolute` to a `Temporal.DateTime` by specifying the desired time zone, add the duration, and then convert it back.
```

Comment:
Were we not going to drop DateTime arithmetic?

Reply:
@pipobscure successfully convinced against dropping it, it was something that had been considered in the early stages of the proposal already. Details in the March 5 meeting minutes (which should be posted to the repo soon — @ryzokuken?)